include/unk_0201E7D8.h

```diff
@@ -0,0 +1,61 @@
+#ifndef POKEDIAMOND_UNK_0201E7D8_H
+#define POKEDIAMOND_UNK_0201E7D8_H
+
+#include "fx.h"
+
+
+struct CameraHistory
+{
+    s32 count;
+    s32 readIdx;
+    s32 writeIdx;
+    s32 writeIdxInit;
+    BOOL overrideEnabled;
+    BOOL enableUpdateX;
+    BOOL enableUpdateY;
+    BOOL enableUpdateZ;
+    VecFx32 * vecs;
+};
+
+struct CameraAngle
+{
+    u16 x;
+    u16 y;
+    u16 z;
+    u16 unk_06;
+};
+
+struct CameraPerspective
+{
+    fx32 fovySin;
+    fx32 fovyCos;
+    fx32 aspect;
+    fx32 n;
+    fx32 f;
+};
+
+struct CameraLookAt
+{
+    VecFx32 camPos;
+    VecFx32 camTarget;
+    VecFx32 camUp;
+};
+
+struct CameraWork
+{
+    struct CameraPerspective perspective;
+    struct CameraLookAt lookAt;
+    fx32 distance;
+    struct CameraAngle angle;
+    u8 perspectiveType;

```

Comment:
this feels like it should be an enum but first of all it's a u8, and I would be very unsure what the values would be, perhaps constants instead?